hangman/test/hangman/game_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule HangmanTest do
```

Comment:
## +4

Yea! Tests!

Ignoring the first one, the next three are interesting, because all they really test is that the code is there and doesn't blow up.

If this is what you intended, then you could probably document the fact, because it otherwise looks a little like the test is unfinished.

Maybe it would be good enough to simply test that there's a valid return value?